### PR TITLE
create RDS-postgres via tf

### PR DESCRIPTION
**Pull request by Frank-Xie-SmartNews**

### CODE REVIEW

File: terraform/prd/tokyo/main.tf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    region = "ap-northeast-1"
+    bucket = "terraform-remote-state-smartnews-prd-ap-northeast-1"
+    key    = "realtime-streaming/realtime-streaming-llm-ops.tfstate"
```
Comment (zheyu001) [inline terraform/prd/tokyo/main.tf:5]: `realtime-streaming` is the system value from ops-bootstrap. If you are using a different system, please use that value.

File: terraform/prd/tokyo/rds.tf [hunk 2]
```diff
@@ -0,0 +1,89 @@
+locals {
+  rds_engine         = "aurora-postgresql"
+  rds_engine_version = "15.3"
+  rds_cluster_identifier = "llm-ops-prd"
+}
+
+resource "aws_db_subnet_group" "rds_tokyo_db_subnet_group" {
+  provider   = aws.primary
+  name       = "dpsh-platform-prd-db-subnet-group"
```
Comment (zheyu001) [inline terraform/prd/tokyo/rds.tf:9]: Use a name specified for llm-ops.

File: terraform/prd/tokyo/rds.tf [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  subnet_ids = [data.aws_subnet.news_private_b_0.id, data.aws_subnet.news_private_c_0.id, data.aws_subnet.news_private_d_0.id]
+
+  tags = {
+    Cost   = var.cost
+    Env    = "production"
+    Name   = "dpsh-platform-prd"
```
Comment (zheyu001) [inline terraform/prd/tokyo/rds.tf:15]: Use llm-ops-prd

File: .circleci/config.yml [hunk 4]
```diff
@@ -431,40 +431,45 @@ workflows:
...
       - terraform/trigger:
           docker: *orb_docker_terraform
-          name: terraform/trigger
+          name: terraform/trigger/dev
           system: realtime-streaming
           path: .
+      - terraform/trigger:
+          docker: *orb_docker_terraform
+          name: terraform/trigger/prd
+          system: realtime-streaming
```
Comment (zheyu001) [inline .circleci/config.yml:445]: Are we planning to deploy using realtime-streaming system or llm-ops system?

File: terraform/prd/tokyo/main.tf (hunk 1 above)
Comment (Frank-Xie-SmartNews) [reply terraform/prd/tokyo/main.tf:5]: This service deploy under common-0 tokyo prd realtime-stremaing, so I guess should use this system?

File: terraform/prd/tokyo/rds.tf [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+resource "aws_rds_cluster" "rds_tokyo_cluster" {
+  provider                    = aws.primary
+  deletion_protection         = true
+  engine                      = local.rds_engine
+  engine_version              = local.rds_engine_version
+  db_subnet_group_name        = aws_db_subnet_group.rds_tokyo_db_subnet_group.name
+  vpc_security_group_ids      = [data.aws_security_group.smartnews.id]
+  cluster_identifier          = "llm-ops-prd"
+  master_username             = "postgres"
+  master_password             = "postgres"
```
Comment (zheyu001) [inline terraform/prd/tokyo/rds.tf:30]: Create a secret manager to store the password like here: https://github.com/smartnews/DP-clickhouse/blob/master/clusters/terraform/module/main.tf#L30-L54 

And refer to the password from random_password.

File: terraform/prd/tokyo/rds.tf [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  database_name               = "pezzo"
+  availability_zones          = ["ap-northeast-1b", "ap-northeast-1c", "ap-northeast-1d"]
+  skip_final_snapshot         = true
+
+  tags = {
+    Cost   = var.cost
+    Env    = "production"
+    Name   = "dpsh-platform-prd"
```
Comment (zheyu001) [inline terraform/prd/tokyo/rds.tf:38]: Please update

File: terraform/prd/tokyo/rds.tf [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  engine_version       = local.rds_engine_version
+  db_subnet_group_name = aws_db_subnet_group.rds_tokyo_db_subnet_group.name
+  db_parameter_group_name = aws_db_parameter_group.dpsh-platform-prd-parameter-group.name
+  promotion_tier       = 0
+
+
+  tags = {
+    Cost   = var.cost
+    Env    = "production"
+    Name   = "dpsh-platform-prd"
```
Comment (zheyu001) [inline terraform/prd/tokyo/rds.tf:61]: Update

File: terraform/prd/tokyo/rds.tf [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  }
+  parameter {
+    name  = "wait_timeout"
+    value = "31536000"
+  }
+  parameter {
+    name  = "temptable_max_mmap"
+    value = "10737418240"
+  }
+}
```
Comment (zheyu001) [inline terraform/prd/tokyo/rds.tf:89]: You don't need this. They are mysql specified.

**Review by zheyu001 — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 4 above)
Comment (Frank-Xie-SmartNews) [reply .circleci/config.yml:445]: realtime-streaming system

File: terraform/prd/tokyo/rds.tf (hunk 2 above)
Comment (Frank-Xie-SmartNews) [reply terraform/prd/tokyo/rds.tf:9]: update to ```dpsh-llm-ops-prd-db-subnet-group```

File: terraform/prd/tokyo/rds.tf (hunk 3 above)
Comment (Frank-Xie-SmartNews) [reply terraform/prd/tokyo/rds.tf:15]: update to ```dpsh-llm-ops-prd```

File: terraform/prd/tokyo/rds.tf (hunk 5 above)
Comment (Frank-Xie-SmartNews) [reply terraform/prd/tokyo/rds.tf:30]: got it, for quick set up, let me create the secret later

File: terraform/prd/tokyo/rds.tf (hunk 6 above)
Comment (Frank-Xie-SmartNews) [reply terraform/prd/tokyo/rds.tf:38]: update to ```dpsh-llm-ops-prd```

File: terraform/prd/tokyo/rds.tf (hunk 7 above)
Comment (Frank-Xie-SmartNews) [reply terraform/prd/tokyo/rds.tf:61]: update to ```dpsh-llm-ops-prd```

File: terraform/prd/tokyo/rds.tf (hunk 8 above)
Comment (Frank-Xie-SmartNews) [reply terraform/prd/tokyo/rds.tf:89]: remove the parameter-group resource

**Review by zheyu001 — APPROVED** [review]

**Pull request merged**